### PR TITLE
Move Kops to its own testgrid dasboard

### PR DESCRIPTION
**Pull request by hakman**
Kops dashboard is becoming pretty busy:
https://testgrid.k8s.io/sig-cluster-lifecycle-kops#Summary

Some order would help with this: https://github.com/kubernetes/test-infra/pull/17435#issuecomment-621346432

/cc @justinsb @rifelpet

### CODE REVIEW

File: config/jobs/kubernetes/kops/OWNERS [hunk 1]
```diff
@@ -15,5 +15,3 @@ approvers:
 reviewers:
   - joshbranham
   - robinpercy
-labels:
-- sig/cluster-lifecycle
```
Comment (rifelpet) [inline config/jobs/kubernetes/kops/OWNERS:19]: I'm wondering if we should keep these. Even if we make kops its own testgrid dashboard, the project is still owned by sig-cluster-lifecycle

Comment (hakman) [reply config/jobs/kubernetes/kops/OWNERS:19]: We should add some labels I guess, but not sure if we want to keep this or add a new one more explicit.

Comment (hakman) [reply config/jobs/kubernetes/kops/OWNERS:19]: Labels are back :)

File: config/tests/testgrids/config_test.go [hunk 2]
```diff
@@ -55,6 +55,7 @@ var (
 	}
 	orgs = []string{
 		"conformance",
+		"kops",
```
Comment (justinsb) [inline config/tests/testgrids/config_test.go:58]: I take it we have to do this to create the dashboards without the prefix?  I'm not sure how controversial this is going to be...

I wonder if we could go into companies instead?  I feel like we can justify being on par with those (e.g. istio).

Comment (justinsb) [reply config/tests/testgrids/config_test.go:58]: cc @BenTheElder what do you think here?

File: config/jobs/image-pushing/k8s-staging-kops.yaml [hunk 3]
```diff
@@ -5,7 +5,7 @@ postsubmits:
       cluster: k8s-infra-prow-build-trusted
       annotations:
         # this is the name of some testgrid dashboard to report to.
-        testgrid-dashboards: sig-cluster-lifecycle-kops
```
[the remark this answers is not in the document]
Comment (hakman) [reply config/jobs/image-pushing/k8s-staging-kops.yaml:8]: Agreed. Done.

**Comment by justinsb** [comment]
This LGTM, question for @BenTheElder on whether we should be a company or an org, but it's easy enough to move around...

/approve
/lgtm

**Comment by BenTheElder** [comment]
Spoke to @spiffxp: for now :man_shrugging: but there may be more enforcement of the organization of testgrid in the future.
/approve

**Comment by hakman** [comment]
Thanks @justinsb & @BenTheElder 😄

**Pull request merged**